sky/shell/platform/ios/framework/Source/accessibility_bridge.h

```diff
@@ -39,12 +36,7 @@ class AccessibilityBridge;
  * The parent of this node in the node tree. Will be nil for the root node and
  * during transient state changes.
  */
-@property(nonatomic, readonly) AccessibilityNode* parent;
-
-/**
- * This node's children in the node tree.
- */
-@property(nonatomic, readonly) NSArray<AccessibilityNode*>* children;
```

Comment:
Yes, although there's actually less manual refcounting in this patch than before because we already needed to manually twiddle the ref counts in several places.  Now all we do is release objects when we remove them from the `_objects` map.